src/atoms/fileActions.test.ts

```diff
@@ -463,4 +471,148 @@ describe('fileActions', () => {
     }
     expect(activeFile.current.data).toEqual({ type: 'tiptap', textContent: 'some content' });
   });
+
+  it('should return the content of the active pane', () => {
+    const store = createStore();
+    const activePane = runGetAtomHook(activePaneContentAtom, store);
+    const focusPanel = runSetAtomHook(focusPaneAtom, store);
+
+    const paneToFocus: PaneId = activePane.current.id === 'LEFT' ? 'RIGHT' : 'LEFT';
+
+    act(() => {
+      focusPanel.current(paneToFocus);
+    });
+
+    expect(activePane.current.id).toBe(paneToFocus);
+  });
+
+  describe('File content atoms', () => {
```

Comment:
nit: maybe we should split this file in multiple files by scope.
For example, you can see that we have two afterEach in this file :-).

Reply:
I agree but I don't know how to name the files, because so far we have been using the convention `fileName.test.ts` to test `fileName.ts`. The issue is that `fileActions.ts` contains a lot of logic. Do you think we should split it too?

For the second `afterEach`, it's because it's in a nested `describe` block, but apparently top-level `afterEach` also runs after each nested test, so you're right, I can remove it

Reply:
I agree but I don't know how to name the files, because so far we have been using the convention fileName.test.ts to test fileName.ts. The issue is that fileActions.ts contains a lot of logic. Do you think we should split it too?

For the second afterEach, it's because it's in a nested describe block, but apparently top-level afterEach also runs after each nested test, so you're right, I can remove it

Reply:
You are right, maybe we should also split `fileActions.ts`. We can make that change in a different PR if you want.